test/ClangImporter/experimental_diagnostics_cmacros.swift

```diff
@@ -3,146 +3,141 @@
 import macros
 
 _ = INVALID_INTEGER_LITERAL_2
-// CHECK:      macros.h:{{[0-9]+}}:{{[0-9]+}}: error: invalid digit 'a' in decimal constant
```

Comment:
This "invalid digit" diagnostic is not controlled by the new system, so I removed it from the test for the time being. It is in no way problematic though, just thought it didn't belong.